### ISSUE TITLE
Worldgen stuck

### ISSUE THREAD

**Issue by LemADEC** [wuppy29/WuppyMods#5]
When using just Forge Mod Loader version 7.10.85.1236 and "Goblins and Giants 1.7.10-1.0.4.3.jar", initial world generation gets stuck while starting a superflat world with Seed: -247458159784279772 or  a normal world with Seed: -3113879159425603667.

**Comment by wuppy29** [comment]
Are you sure it actually freezes? Because Goblins and Giants adds a new
dimension and a bunch of new stuff to the overworld. All this extra
generation can greatly slow down world creation.
- Wuppy

**Comment by LemADEC** [comment]
Yes, the game stop doing the initial spawn generation, there's no logs and 0% CPU for several minutes until I kill the application with task manager.

**Comment by wuppy29** [comment]
Very strange. I doubt I can fix that though, it really seems like a Minecraft or forge issue.
Lets hope this issue is gone in 1.8.

**Comment by LemADEC** [comment]
Did you try running those worldgen in your debug environment?

**Comment by wuppy29** [comment]
Yeah, usually it works just fine. Sometimes it will pause though like you
said. However, when you reenter that world it will generate just fine.

**Comment by LemADEC** [comment]
I've tried multiple times to reenter the world and it restart the spawn worldgen and gets stuck again.
